Chummer/Backend/Equipment/Armor.cs

```diff
@@ -856,6 +856,7 @@ public void Print(XmlTextWriter objWriter, CultureInfo objCulture, string strLan
             objWriter.WriteElementString("category", DisplayCategory(strLanguageToPrint));
             objWriter.WriteElementString("category_english", Category);
             objWriter.WriteElementString("armor", DisplayArmorValue);
+            objWriter.WriteElementString("armorcapacity", ArmorCapacity);
```

Comment:
I think you're printing the wrong `ArmorCapacity`. `ArmorCapacity` is the string that Chummer uses internally as a baseline to calculate `TotalArmorCapacity`, which is probably what you want. However, `TotalArmorCapacity` is also used to calculate strings for the calculated capacity and the amount of capacity remaining, so you might want to export those, too. Also, overall, both `TotalArmorCapacity` and the calculated capacity output strings that are based on the decimal formatting of the current UI language. Because the printed language can be different from the UI language, you'd also probably want to change and/or shift code around to let you get versions of those properties that are formatted in any language you'd want instead of in just the UI language.

Reply:
This is great feedback. I've been looking through this code for a bit trying to absorb the difference between the member variables and properties in both the armor and armormod files.

I'll go back to the code and review what it is that I want to print in the JSON

Reply:
Also, side note, but Contributors.cs is not a file you're supposed to edit manually, even if you're fetching info from the page you linked. It's updated automatically via a T4 macro that uses GitHub's API (and a private access token that is set up on the AppVeyor build machine that's used to create official builds).

Reply:
Thank you for the clarification. I've never used a T4 template before, and didn't want to remove someone's name from contribution.

Is this something that should be updated in the Contributing to Chummer Wiki?

Reply:
Also, the file was edited by running the template in Visual Studio, only to make mention that I did not edit it manually

Reply:
`CapacityRemaining` is a decimal, so that can be printed out with the correct decimal place mark. `TotalArmorCapacity` looks like it's only ever a string that looks like an integer. Unless the capacity somehow is a decimal, I believe I can just print `TotalArmorCapacity` as it's basically language agnostic